espresso-server/settings.gradle.kts

```diff
@@ -0,0 +1 @@
+include(":app")
```

Comment:
This file was renamed to this in https://github.com/appium/appium-espresso-driver/blob/master/package.json#L39 so I guess this was expected

Reply:
cc @misa